packages/devtools_app/lib/src/inspector/flutter/inspector_service_flutter_extension.dart

```diff
@@ -111,84 +135,120 @@ extension InspectorFlutterService on ObjectGroup {
+                      constraintsProperty.addAll(<String, Object>{
+                        'minWidth': constraints.minWidth.toString(),
+                        'minHeight': constraints.minHeight.toString(),
+                        'maxWidth': constraints.maxWidth.toString(),
+                        'maxHeight': constraints.maxHeight.toString(),
+                      });
                     }
-                    if (renderObject is RenderBox) {
-                      additionalJson['size'] = <String, Object>{
-                        'width': renderObject.size.width.toString(),
-                        'height': renderObject.size.height.toString(),
-                      };
-  
-                      final ParentData parentData = renderObject.parentData;
-                      if (parentData is FlexParentData) {
-                        additionalJson['flexFactor'] = parentData.flex;
-                        additionalJson['flexFit'] =
-                            describeEnum(parentData.fit ?? FlexFit.tight);
-                      }
+                    additionalJson['constraints'] = constraintsProperty;
+                  }
+                  if (renderObject is RenderBox) {
+                    additionalJson['size'] = <String, Object>{
+                      'width': renderObject.size.width.toString(),
+                      'height': renderObject.size.height.toString(),
+                    };
+
+                    final ParentData parentData = renderObject.parentData;
+                    if (parentData is FlexParentData) {
+                      additionalJson['flexFactor'] = parentData.flex;
+                      additionalJson['flexFit'] =
+                          describeEnum(parentData.fit ?? FlexFit.tight);
                     }
                   }
-                  return additionalJson;
                 }
+                return additionalJson;
+              }
             ),
           );
         }
         return Future<Map<String, Object>>.value(<String, Object>{
           'result': result,
         });
+      ''',
+  );
+
+  static final tweakFlexFit = RegistrableServiceExtension(
+    name: 'tweakFlexFit',
+    statements: '''
+      final String id = parameters['id'];
+      final FlexFit flexFit =
+          toEnumEntry<FlexFit>(FlexFit.values, parameters['flexFit']);
+      dynamic object = WidgetInspectorService.instance.toObject(id);
+      if (object == null) return null;
+      final render = object.renderObject;
+      final parentData = render.parentData;
+      if (parentData is FlexParentData) {
+        parentData.fit = flexFit;
+        render.markNeedsLayout();
       }
-      return WidgetInspectorService.instance.registerServiceExtension(
-        name: '$serviceExtensionNameGetLayoutExplorerNode',
-        callback: callback,
+    ''',
+    requireEnumDeserialization: true,
+  );
+  static final tweakFlexFactor = RegistrableServiceExtension(
```

Comment:
nit: rename from tweakFlexFactor to setFlexFactor

Reply:
should I do the same for `tweakFlexFit` and `tweakFlexProperties`?